Form

KevinRoig opened the pull request.

KevinRoig committed:
some changes on form

KevinRoig committed:
form can now browse and upload photos directly on the computer instead of URLs, and select a category

KevinRoig committed:
form can now browse and upload photos directly on the computer instead of URLs, and select a category

KevinRoig committed:
form can now browse and upload photos directly on the computer instead of URLs, and select a category

KevinRoig committed:
form can now browse and upload photos directly on the computer instead of URLs, and select a category

RomainClair reviewed (changes requested):
The upload directory should probably be in the gitignore and all the test images must not be in the git repository

KevinRoig committed:
changes on Vich's path, uploaded images in gitignore

KevinRoig committed:
Merging dev

Merge branch 'dev' of github.com:WildCodeSchool/tours-php-2009-project3-amon into form

KevinRoig committed:
deleting weird photos

KevinRoig committed:
deleting weird photos

KevinRoig committed:
deleting weird photos

KevinRoig committed:
fixing photos bug
Merge branch 'form' of github.com:WildCodeSchool/tours-php-2009-project3-amon into form

KevinRoig committed:
Delete kisspng-handshake-clip-art-mano-5b3ba9c4510dd4.301697211530636740332.png

Delete because error

KevinRoig committed:
Delete logo(1).jpg

Delete because error

KevinRoig committed:
Delete shaun.jpg

Delete because error

KevinRoig committed:
Delete unnamed.png

Delete because error
